components/header/src/Header.tsx

```diff
@@ -106,22 +109,30 @@ export const Header: FC<HeaderProps> = ({
     forceExternal: true
   }];
 
+  // Use the CrownLogo or CoatLogo by default.
+  const crownLogo: ReactNode = <CrownLogo focusable="false" className={classes('logotype')} height="30" width="148" />
+  const coatLogo: ReactNode = <CoatLogo aria-hidden="true" focusable="false" className={classes('logotype', ['coat'])} height="30" width="36" />
+  let headerLogo: ReactNode | null | undefined = govUK ? crownLogo : coatLogo
+
+  // If the logo prop is null or a valid ReactNode, use that instead of the default CrownLogo or CoatLogo component
+  if (_logo === null || isValidElement(_logo)) {
+    headerLogo = _logo
+  }
+
   return (
     <header {...attrs} className={classes()} data-module="govuk-header">
       <WidthContainer maxWidth={maxContentsWidth} className={classes('container', department)}>
         <div className={classes('logo')}>
           <A href={orgHref} classModifiers={[ 'homepage', (orgText && orgText.length > 9) ? 'small' : undefined ]}>
             {
               govUK
-              ? (
-                <CrownLogo focusable="false" className={classes('logotype')} height="30" width="148" />
-              )
+              ? headerLogo
```

Comment:
If the above suggestion was accepted:
```suggestion
              ? logo
```